### ISSUE TITLE
Yahoo changed their returned field values breaking ffn.get() for Yahoo

### ISSUE THREAD

**Issue by bengebre** [pmorissette/ffn#34]
pandas-datareader 0.7.0 (released today) fixes the latest Yahoo data breakage, but Yahoo is now using new field values which breaks ffn.get().  For instance, 'Adj Close' changed to 'adjclose'.  I replaced the two references to 'Adj Close' with 'adjclose' in data.py and now ffn.get() returns data, but the dataframe is only number indexed rather than date indexed.  It's my first day with ffn so I'm not sure what typical behavior is, but I was just comparing my results to the examples and saw this discrepancy.

**Comment by bengebre** [comment]
Followup: I read an Issue regarding pandas-datareader's 0.7.0 stable release fixing the Yahoo problems, but I was actually using the development version pulled straight from github.  Apologies on the confusion.  The issue may be exactly the same when 0.7.0 is released, but it's probably smart to wait to see what the stable version looks like.  It should be within the next week according to this:

https://github.com/pydata/pandas-datareader/issues/495

**Comment by rubik** [comment]
I have temporarily fixed this with the following code:
```python
import pandas_datareader as pdr
import fix_yahoo_finance as yf
pdr.data.get_data_yahoo = lambda *a, **kw: yf.download(*a, **kw, progress=False)
import ffn
```
You will need the `fix_yahoo_finance` package.

**Comment by JordanPlatts** [comment]
Do you want to submit a pull request?

**Comment by JordanPlatts** [comment]
The pull request doesn't work with python 2.7. 

I don't really want to merge it because some people use 2.7 and this seems like a temporary fix till they release pandas_datareader 0.7.0.
